tests/oscillating_collimator_let.py

```diff
@@ -24,7 +25,7 @@
         },
     },
 ]
-TEST_MODES = [TestModes.DEVSIM]
+TEST_MODES = [TestModes.RECSIM]
```

Comment:
we might be able to run these tests in devsim and recsim, but just run the ones that require digital input signals in recsim

Reply:
As far as I know, "recsim" and "devsim" are actually the same thing for a galil?